lib/pos_mempool_test.go

```diff
@@ -0,0 +1,323 @@
+package lib
+
+import (
+	"fmt"
+	"github.com/emirpasic/gods/sets/treeset"
+	"github.com/stretchr/testify/require"
+	"math/rand"
+	"sort"
+	"testing"
+	"time"
+)
+
+func TestTreeSet(t *testing.T) {
+	set := treeset.NewWithIntComparator(7, 10, 15, 22, 13, 200, 5, 0)
+	it := set.Iterator()
+	for it.Next() {
+		fmt.Printf("%v ", it.Value())
+	}
+	fmt.Println()
+
+	set.Remove(22)
+
+	it = set.Iterator()
+	for it.Next() {
+		fmt.Printf("%v ", it.Value())
+	}
+	fmt.Println()
```

Comment:
let's remove print statements? or use some asserts to make this test more useful.